MPF.Frontend/Tools/SubmissionGenerator.cs

```diff
@@ -698,8 +698,8 @@ private static bool ProcessSystem(SubmissionInfo info, RedumpSystem? system, Dri
                     if (isDiscImageCreator)
                         info.CommonDiscInfo!.EXEDateBuildDate = DiscImageCreator.GetPlayStationEXEDate($"{basePath}_volDesc.txt", drive?.GetPlayStationExecutableName());
 
-                    if (!info.CommonDiscInfo!.CommentsSpecialFields!.TryGetValue(SiteCode.InternalSerialName, out string? kp2Exe) || string.IsNullOrEmpty(kp2Exe))
-                        info.CommonDiscInfo.Region = Drive.GetPlayStationRegion(kp2Exe);
+                    if (info.CommonDiscInfo!.CommentsSpecialFields!.TryGetValue(SiteCode.InternalSerialName, out string? kp2Exe) || !string.IsNullOrEmpty(kp2Exe))
```

Comment:
Each of these `||` need to be `&&` for safety

Reply:
Done.